test/integration/marked-spec.js

```diff
@@ -1,5 +1,62 @@
 var marked = require('../../marked.min.js');
 
-it('should run the test', function () {
-  expect(marked('Hello World!')).toBe('<p>Hello World!</p>\n');
+describe('integration suite', function () {
+  it('should put plaintext into a paragraph', function () {
+    expect(marked('Hello World!')).toBe('<p>Hello World!</p>\n');
+  });
```

Comment:
We don't need to keep this test if we don't want to. I just added it as an example

Reply:
Eventually it'll go under a describe('plaintext') or something.